datasets/nasa-usgs-europa-observations.yaml

```diff
@@ -0,0 +1,39 @@
+Name: NASA / USGS Europa Controlled Observations
+Description: 
+Documentation: https://stac.astrogeology.usgs.gov/docs/data/jupiter/europa/galileo_individual_images/
+Contact: https://answers.usgs.gov/
+ManagedBy: "[United States Geological Survey](https://www.usgs.gov)"
+UpdateFrequency: No future updates planned.
+Tags:
+  - planetary
+  - satellite imagery
+  - stac
+  - cog
+License: [CC0 1.0](https://creativecommons.org/publicdomain/zero/1.0/)
+Citation: https://doi.org/10.5066/P9VKKK7C
+Resources:
+  - Description: Scenes and metadata
+    ARN: arn:aws:s3:::astrogeo-ard/*
```

Comment:
bucket is not public

Reply:
I tried to update the policy using the permissions I have. No joy. Pinged my NASA PoC to request that they make the bucket public. I'll update when that happens.

Reply:
Thanks you can also point them to our handbook which references the Cloud Formation template in Step 5. That would simplify the set up and ensure that the bucket policy is set accordingly. Feel free to follow up via email if easier. https://assets.opendata.aws/aws-onboarding-handbook-for-data-providers-en-US.pdf

Reply:
@jeffejefe This should be good to go now!